Binance Detect Moonings.py

```diff
@@ -154,7 +170,7 @@ def wait_for_price():
 
         threshold_check = (-1.0 if min_price[coin]['time'] > max_price[coin]['time'] else 1.0) * (float(max_price[coin]['price']) - float(min_price[coin]['price'])) / float(min_price[coin]['price']) * 100
 
-        # each coin with higher gains than our CHANGE_IN_PRICE is added to the volatile_coins dict if less than MAX_COINS is not reached.
+        # each coin with higher gains than our CHANGE_IN_PRICE is added to the volatile_coins dict if less than TRADE_SLOTS is not reached.
```

Comment:
Ensure tradeslots renamed in the entire script

Reply:
This is consistently confusing to new users, an we meet in the middle and come up with a better word for MAX_COINS? 🙂